### ISSUE TITLE
Infinite Scroll: infinite-scroll class should be added by JS, not PHP

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#743]
This example code appears on http://jetpack.me/support/infinite-scroll/ to show developers how to hide their standard page navigation on pages with infinite scroll 

``` css
.infinite-scroll #navigation,
.infinite-scroll.neverending #footer {
     display: none;
}
```

this is all fine and dandy, but if the infinite scroll client side JS is, for some reason, not working, then users without JS would not be able to able to find the next page since the `infinite-scroll` class is added by the PHP.  
In short, while infinite scroll is enabled by the client side JS, the class is added to the body by the PHP. This should be a relatively easy fix. 

Original trac ticket:
- https://plugins.trac.wordpress.org/ticket/2000
